### PR TITLE
Reorganised

### PR DESCRIPTION
**Pull request by damonge**
- Moved growth likelihoods to subdirectory
- Created module for unified cobaya likelihood

### CODE REVIEW

File: ClLike/setup.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+setup(
+    name="cl_like",
+    version="0.0",
+    description="Multi-tracer angular C_ell likelihood",
+    zip_safe=True,
+    packages=find_packages(),
+    python_requires=">=3.5",
+    install_requires=[
+        "cobaya>=3.0",
+        "sacc>=0.4.2",
```
Comment (carlosggarcia) [inline ClLike/setup.py:12]: I think we should include here `pyccl`, `scipy` and `numpy` (something else?)

File: papers/growth_history/Cobaya/xCell_lkl/ccl.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
+"""
```
Comment (carlosggarcia) [inline papers/growth_history/Cobaya/xCell_lkl/ccl.py:1]: Maybe this filed should have been `git mv`'ed, instead of recreated, as all the others.

**Review by carlosggarcia — CHANGES_REQUESTED** [review]
Two minor comments.

File: papers/growth_history/Cobaya/xCell_lkl/ccl.py (hunk 2 above)
Comment (damonge) [reply papers/growth_history/Cobaya/xCell_lkl/ccl.py:1]: I did `git mv` it...

File: ClLike/setup.py (hunk 1 above)
Comment (damonge) [reply ClLike/setup.py:12]: Done

**Comment by damonge** [comment]
Back to you @carlosggarcia

File: papers/growth_history/Cobaya/xCell_lkl/ccl.py (hunk 2 above)
Comment (carlosggarcia) [reply papers/growth_history/Cobaya/xCell_lkl/ccl.py:1]: Maybe you modified it after `git mv`ing it and without having committed. Whatever.

**Review by carlosggarcia — APPROVED** [review]
LGTM

**Pull request merged**